app/page.jsx

```diff
@@ -1,16 +1,35 @@
 import Link from 'next/link'
 import Header from '@/components/Header'
+import { getTasks } from '@/lib/routes'
+
+export default async function Home() {
+  const tasks = await getTasks()
+  console.log(tasks)
 
-export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center">
```

Comment:
i think we should put these classes on the body "flex min-h-screen flex-col items-center" and the max-width ("max-w-3xl") on main since we want everything, including the header to have a max-width and be centered. what do you think?